src/components/resource-evaluation-comment/ResourceEvaluationComment.js

```diff
@@ -13,6 +13,7 @@ const ResourceEvaluationComment = ({ evaluation }) => {
           stars={5}
           cancel={false}
           readOnly="true"
+          style={{ width: '100%' }}
```

Comment:
Inline style should be avoided

Reply:
It is because is the only way to modify primereact components. There is no alternative.